win32/updateBuildLibs.py

```diff
@@ -265,6 +251,9 @@ def main():
         usage()
 
     print('Updating source by %s branch.' % branch)
+    if not os.path.exists(LOG_PATH):
+        os.makedirs(LOG_PATH)
+
```

Comment:
I think that it is great to get rid of the code duplication for creating the log files directory. However, why are we executing this code after we have already written log files to the directory, which will ostensibly cause it to be created? To me, it seems more logical to create the log directory before we start writing log files; as it is, I expect that unless there was a problem creating the log files above,  the os.path.exists(LOG_PATH) call will always return true, so this code "feels" wrong.